x-pack/plugins/data_enhanced/server/search/request_utils.ts

```diff
@@ -58,7 +61,6 @@ export function getDefaultAsyncGetParams(): Pick<
   'keep_alive' | 'wait_for_completion_timeout'
```

Comment:
```suggestion
  'wait_for_completion_timeout'
```

Reply:
Added a smarter version that still polls and extends if the search is not run within the context of a session